### PR TITLE
 Modification documents

### PR DESCRIPTION
**Pull request by cqqqq777**
Updated Chinese documents and corrected English documents

### CODE REVIEW

**Review by cqqqq777 — COMMENTED** [review]
Updated Chinese documents and corrected English documents

**Review by cqqqq777 — COMMENTED** [review]
Chinese documents are based on English documents

File: docs/API.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ func main() {
 
 | Bucket operations                                     | Object operations                                   | Presigned operations                          | Bucket Policy/Notification Operations                         | Client custom settings                                |
 | :---                                                  | :---                                                | :---                                          | :---                                                          | :---                                                  |
-| [`MakeBucket`](#MakeBucket)                           | [`GetObject`](#GetObject)                           | [`PresignedGetObject`](#PresignedGetObject)   | [`SetBucketPolicy`](#SetBucketPolicy)                         | [`SetAppInfo`](#SetAppInfo)                           |
+| [`MakeBucket`](#Ma[API.md](zh_CN%2FAPI.md)keBucket)                           | [`GetObject`](#GetObject)                           | [`PresignedGetObject`](#PresignedGetObject)   | [`SetBucketPolicy`](#SetBucketPolicy)                         | [`SetAppInfo`](#SetAppInfo)                           |
```
Comment (harshavardhana) [inline docs/API.md:63]: There should be no mention of Chinese docs in English

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by jiuker** [comment]
@cqqqq777 Modify your pr plz.

**Comment by harshavardhana** [comment]
Stale PR closing.

**Pull request closed without merging**